src/TLuaInterpreter.cpp

```diff
@@ -12517,6 +12521,14 @@ void TLuaInterpreter::initIndenterGlobals()
     luaL_dostring(pIndenterState, QStringLiteral("package.cpath = package.cpath .. ';%1/lib/?.so'")
                   .arg(QCoreApplication::applicationDirPath())
                   .toUtf8().constData());
+#elif defined(Q_OS_WIN32)
+    // For Qt Creator builds, add search paths one and two levels up from here, then a 3rdparty directory:
```

Comment:
Is the order here as you intend?

As it is, it goes "add search paths in a 3rdparty directory up two levels and across from here; directly below here and then in a 3rdparty directory up one level and across from here", the first and the last handle shadow building in qt where `CONFIG` either contains `debug_and_release_target` or does not. **This is not the same as the `debug_and_release` option** and it controls whether qmake (helpfully) appends an extra `/debug` or `/release` onto the end of the **build** directory (when the option is present and it **is** by default) even if that is not helpful because you are only building one of them...  The remaining (middle case here) is for the assembled binary "_packaged_" case where all the libraries and everything has been put into the same directory as the executable except for the external Mudlet/Geyser/LuaCodeFormatter files where are in subdirectories directly beneath there.

BTW It took me a while to figure out that the `.` character in a lua file name can also be a directory path separator so that the line below:

    require('lcf.workshop.base')

actually means:

    require('lcf/workshop/base.lua')

😮